README.md

```diff
@@ -1 +1,22 @@
-# hw01
+# Homework 1: Markdown Biography
+
+## Introduction
+
+![headshot](https://media-exp1.licdn.com/dms/image/C4E03AQE5isHwmTn2mw/profile-displayphoto-shrink_200_200/0?e=1605139200&v=beta&t=h21NrPkRVwGyF5AosXS3Fn6asKrGUhBjrKMWZEv0mr0)
+
+My name is **Kaitlyn Van Baalen**. I am a third year Public Policy major and Statistics minor at the college.
+
+## Interests
+
+Some of my interests include:
+* Running - I am a part of the varsity cross country and track teams at the university. I focus on a few events:
+  * 800 meters
+  * 1500 meters
+  * One mile
+* Education - I have worked as a tutor in many different subjects and for several different educational nonprofits, because I have a passion for equitable education.
+* Reading - I love a good book of almost any genre, but fantasy and nonfiction memoirs are some of my particular favorites.
+* Politics - When I'm not listening to an audiobook, I'm probably listening to a political podcast. I try to make some time each week for political volunteering.
+* Travel - Whenever it is safe to travel again, I have an evergrowing list of places I'd love to go.
+
+## Goals
+I'm excited to take CFSS because I've learned done data analysis in my statistics classes, but I want to know *how* to get the data to the point where it can be analyzed. In the real world, I know it isn't as simple as downloading a pristine file from Canvas. I think this will be useful for me as I hope to work in policy research in the future.
```

Comment:
Great idea including a goals section in your bio, I like that you used lists in the interests section and wrote out your goals in a paragraph.